paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -148,8 +146,55 @@ internal class CustomerSheetViewModel(
         scope = viewModelScope.plus(workContext)
     )
 
+    private val isEditing = MutableStateFlow(false)
+    private val selectionConfirmationState = MutableStateFlow(
+        SelectionConfirmationState(
+            isConfirming = false,
+            error = null,
+        )
+    )
+    private val customerState = MutableStateFlow(
+        CustomerState(
+            paymentMethods = listOf(),
+            configuration = configuration,
+            currentSelection = originalPaymentSelection,
+            metadata = null,
+        )
+    )
```

Comment:
We should able to create sub-flows for `SelectPaymentMethodInteractor` with these values